test/e2e/specs/plugins-api.test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Internal dependencies
+ */
+import '../support/bootstrap';
+import { newPost, newDesktopBrowserPage, toggleMoreMenuItem } from '../support/utils';
+import { activatePlugin, deactivatePlugin } from '../support/plugins';
+
+describe( 'Using Plugins API', () => {
+	beforeAll( async () => {
+		await newDesktopBrowserPage();
+		await activatePlugin( 'gutenberg-test-plugin-plugins-api' );
+		await newPost();
+	} );
+
+	afterAll( async () => {
+		await newDesktopBrowserPage();
+		await deactivatePlugin( 'gutenberg-test-plugin-plugins-api' );
+	} );
+
+	it( 'Should open plugins sidebar using More Menu item and render content', async () => {
+		await toggleMoreMenuItem( 'My title plugin' );
+
+		const pluginSidebarContent = await page.$eval( '.edit-post-sidebar', ( el ) => el.innerHTML );
+		expect( pluginSidebarContent ).toMatchSnapshot();
```

Comment:
Not sure how I feel about snapshotting the whole sidebar? Can't we target something more specific, for example, check that the title changes with the post title in the editor?

Reply:
Yes, I will open another PR soon :)
I want more tests!!!